python/lsst/ts/cbp/simulator/__init__.py

```diff
@@ -0,0 +1,156 @@
+from lewis.devices import StateMachineDevice
```

Comment:
Are you sure you want to do all this coding in  an `__init__` file?

Please, consider moving this to a module and have a clean `__init__`